developer/tutorial/tutorial-debugging-server-code.md

```diff
@@ -0,0 +1,121 @@
+# Tutorial: Debugging server-side code
+
+As of Meteor 1.6, you can now debug server code using the `--inspect` flag in Chrome DevTools and within code editors like [Visual Studio Code (VS Code)](https://code.visualstudio.com/), a free code editor. Here are the steps to get started:
+
+## Launch the application in `inspect` mode
+
+1. Before we get started, make sure you have at least Reaction 1.6. You can do that by running:
+
+```sh
+reaction -v
+```
+
+You should see something like:
+```sh
+Node: 8.2.1
+NPM: 5.6.0
+Meteor Node: 8.9.3
+Meteor NPM: 5.5.1
+Reaction CLI: 0.24.2
+Reaction: 1.6.4
+Reaction branch: master
+Docker: 17.09.0-ce
+```
+
+If your Reaction version is older than 1.6, you will have to upgrade to at least 1.6 first.
+
+2. Instead of using `reaction` to run the application, use the inspector mode:
```

Comment:
I think we need to mention that you need to run `reaction` at least once for plugin imports to be set up. Also I think you want to pass in `--settings` and maybe `--raw-logs` to be equivalent to the `reaction` command

Reply:
Got it.